library.py

```diff
@@ -0,0 +1,43 @@
+from typing import Optional
+from library.model.book import Book
+from library.model.genre import Genre
+
+
+class Library():
+
+    # Prints crime audio books
```

Comment:
Coudl omit that comment - the function name already makes it clear. And it is not 100% accurate (prints longes book actually)